hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1727,6 +1727,15 @@ possible configurations would overwhelm and obscure the important.
       ThreadPool.
     </description>
   </property>
+  <property>
+    <name>hbase.http.enable.prometheus.servlets</name>
+    <value>false</value>
+    <description>
+      Enable prometheus servlets /prom and /prom2 for prometheus based monitoring.
+      /prom is based on new HBase metrics API and all metrics are not exported for now.
+      /prom2 is based on the old hadoop2 metrics API and has all the metrics.
```

Comment:
Why not /prom and /prom-old ? Why have the old at all (maybe you want to backport this)? When /prom and /prom2, w/o close reading, users will think they need to read from /prom2 because 2 is a later number than no number?

Also, can you say more on old vs new metrics. I'm not clear. Would be good to also clean up the description in here so clear to the casual reader.

Thakns for adding the flag.

Reply:
Is 'new metrics' our use of 'hadoop metrics', a move we made years ago? What versions in release do the old way of metrics? Thanks.

Reply:
@saintstack 
> Why not /prom and /prom-old ? Why have the old at all (maybe you want to backport this)? When /prom and /prom2, w/o close reading, users will think they need to read from /prom2 because 2 is a later number than no number?
> 
> Also, can you say more on old vs new metrics. I'm not clear. Would be good to also clean up the description in here so clear to the casual reader.
> 
> Thakns for adding the flag.

That makes sense, I will change it to /prom-old or similar and fix the documentation of the flag. (Answers to other questions are in the next block)



> Is 'new metrics' our use of 'hadoop metrics', a move we made years ago? What versions in release do the old way of metrics? Thanks.

HBASE-9774 brought in hbase-native metrics collection for the coprocessors and a decision was made to to use this API to record all the other metrics as well (hbase-metrics-api / README.txt). I am referring to this as "new API'. Now all the work to consume this new API isn't done so we still have to depend on the "old", "hadoop2" based metric collection.

Reply:
Thanks. Now I see where you are coming from.